pages/express-relay/integrate-as-searcher/svm.mdx

```diff
@@ -0,0 +1,195 @@
+import { Callout, Tabs, Steps } from "nextra/components";
+
+# SVM Searcher Integration
+
+SVM Express Relay searchers fulfill opportunities representing limit orders on the [Limo]() program.
+
+<Steps>
+
+### Subscribe to New Opportunities
+
+Express Relay provides searchers with [Typescript](https://github.com/pyth-network/per/tree/main/sdk/js) and [Python](https://github.com/pyth-network/per/tree/main/sdk/python) SDKs to interact with Express Relay.
+Searchers can also directly fetch available opportunities via HTTP or subscribe to them via WebSocket.
+
+<Tabs items={['Typescript', 'Python', 'HTTP', 'Websocket']}>
+
+<Tabs.Tab>
+Pyth provides a Typescript SDK, which allows searchers to subscribe to opportunities:
+
+```typescript
+import { Client, Opportunity } from "@pythnetwork/express-relay-js";
+
+const handleOpporunity = async (opportunity: Opportunity) => {
+  // Implement your opportunity handler here
+};
+
+const client = new Client(
+  { baseUrl: "https://pyth-express-relay-mainnet.asymmetric.re" },
+  undefined, // Default WebSocket options
+  handleOpporunity
+);
+await client.subscribeChains(["development-solana"]);
+```
+
+</Tabs.Tab>
+<Tabs.Tab>
+Pyth provides a Python SDK, which allows searchers to subscribe to opportunities:
+
+```python copy
+from express_relay.client import (
+    ExpressRelayClient,
+)
+from express_relay.express_relay_types import Opportunity
+
+def opportunity_callback(opportunity: Opportunity):
+    # Implement your opportunity handler here
+    pass
+
+client = ExpressRelayClient(
+    'server_url',
+    None,
+    opportunity_callback,
+    None,
+)
+await client.subscribe_chains(['development-solana'])
+```
+
+</Tabs.Tab>
+<Tabs.Tab>
+Searchers can request opportunities through an HTTP **GET** call to the [`/v1/opportunities`](https://pyth-express-relay-mainnet.asymmetric.re/docs#tag/opportunity/operation/get_opportunities) endpoint.
+
+```bash copy
+curl -X 'GET' \
+  'https://pyth-express-relay-mainnet.asymmetric.re/v1/opportunities?chain_id=development-solana&mode=live'
+```
+
+Opportunities are short-lived and could be executed in a matter of seconds. So, the above endpoint could return an empty response.
+
+</Tabs.Tab>
+<Tabs.Tab>
+Searchers can connect to the server via WebSocket to reduce latency and subscribe to various events. The WebSocket endpoint lives at `/v1/ws`(e.g `wss://pyth-express-relay-mainnet.asymmetric.re/v1/ws`).
+Here is a sample JSON payload to subscribe to opportunities:
+
+```bash copy
+{
+  "id": "1",
+  "method": "subscribe",
+  "params": {
+    "chain_ids": ["development-solana"]
+  }
+}
+```
+
+Consult [`Websocket API reference`](./websocket-api-reference.mdx) for a complete list of methods and parameters.
+
+</Tabs.Tab>
+</Tabs>
+
+The server responds with opportunities in the following format:
+
+```bash copy
+{
+  "order": "DUcTi3rDyS5QEmZ4BNRBejtArmDCWaPYGfN44vBJXKL5",              // The Limo order to be executed, encoded in base64
```

Comment:
this is a bit unrealistic value and it's confusing since it's identical to the other one :D maybe something a bit longer with ... in the end?